### PR TITLE
multi_nics_verify: Make sure qemu process exits

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID: 1743092

### CODE REVIEW

File: qemu/tests/multi_nics_verify.py [hunk 1]
```diff
@@ -103,3 +103,4 @@ def check_nics_num(expect_c, session):
         nic_interface.append(guest_ip)
     session_srl.close()
     logging.info("All the [ %s ] NICs get IPs." % nics_num)
+    vm.destroy()
```
Comment (PaulYuuu) [inline qemu/tests/multi_nics_verify.py]: Drop `kill_vm = no` in multi_nics_verify.cfg

**Comment by PaulYuuu** [comment]
1. Don not use bug id in summary
2. Drop test result in commit message, it should be a comment
3. Rebase to latest master branch, and then make your changes, this bug only needs one commit.

File: qemu/tests/multi_nics_verify.py (hunk 1 above)
Comment (yanglei-rh) [reply qemu/tests/multi_nics_verify.py]: ok,I will delete it.

**Comment by PaulYuuu** [comment]
> ID: 1743092

The reason for adding the modified content here is better(can copy the commit message), other LGTM.
Acked-by: Yihuang Yu <yihyu@redhat.com>

**Comment by pezhang** [comment]
LGTM.

Acked by: Pei Zhang pezhang@redhat.com

**Comment by vivianQizhu** [comment]
> > ID: 1743092
> 
> The reason for adding the modified content here is better(can copy the commit message), other LGTM.
@yanglei-rh Next time please pay attention to this, thanks.

> Acked-by: Yihuang Yu [yihyu@redhat.com](mailto:yihyu@redhat.com)

**Pull request merged**